src/Tribe/Views/V2/View.php

```diff
@@ -863,6 +863,72 @@ public function prev_url( $canonical = false, array $passthru_vars = [] ) {
 		return $url;
 	}
 
+	/**
+	 * Returns the top bar navigation label associated with this View logical, next view.
+	 *
+	 * @since TBD
+	 *
+	 * @return string
+	 */
+	public function top_bar_next_label() {
```

Comment:
In place of filtering into this method directly, I'd rather go with the same pattern we used for `View::setup_template_vars` or `View::setup_repository_arguments`.

Break this method down into two methods, I'm using the `View::top_bar_next_level` method as an example:

1. `View::top_bar_next_label` is the method that produces the default label ("Next")
2. `View::filter_top_bar_next_label` is the method that filters the label

In a case like the Day View, only the first method would be overridden, not the second. This de-clutters the code, slims it down and keeps the filters in one place only.